src/components/Layout/NavigationBar/NavigationBar.tsx

```diff
@@ -155,16 +152,30 @@ const NavLogo = () => {
   );
 };
 
-export const NavigationBar: FunctionComponent<{
+const flagName = "MAGIC_DEMO";
+const identityFn = (x: any) => x;
+interface NavigationBarProps {
   toggleNavBar: boolean;
   setToggleNavBar: (toggleNavbar: boolean) => void;
-}> = (props) => {
+  leftItems: NavigationItem[];
+  rightItems: NavigationItem[];
+}
+
+export const NavigationBar: FunctionComponent<NavigationBarProps> = (props) => {
+  const { leftItems, rightItems } = props;
+  const [derivedLeftItems, setDerivedLeftItems] = React.useState<NavigationItem[]>([]);
+  const flag = isDevelopment ? Features[flagName].development : Features[flagName].production;
```

Comment:
dunno if i make sense not, i think need that hook to overwrite via localStorage console, so the dev can show

```suggestion
  const flagToggle = isDevelopment ? Features[flagName].development : Features[flagName].production;
  const { getFeatureFlagOverride } = useFeatureFlagOverride();
  const flag = getFeatureFlagOverride(flagName) ?? flagToggle;
```

Reply:
something liek `localStorage.setItem("FEATURE_FLAG", JSON.stringify({ MAGIC_DEMO: true }));` in dev console